<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow
{
	background-color: #292826;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>741</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_Video">
       <property name="title">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>621</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QSlider" name="horizontalSlider_Duration">
       <property name="styleSheet">
        <string notr="true">QSlider::groove:horizontal {
        background-color: #91908d;
        border: 1px solid #000000;
        height: 4px;
        border-radius: 3px;
}

QSlider::handle:horizontal {
	background-color: #58d162;
	border: 2px solid #515451;
	margin: -7px 0;
	width: 5px;
	border-radius: 4px;
}

QSlider::add-page:horizontal {
	background-color: #91908d;
     border-radius: 3px;
}

QSlider::sub-page:horizontal {
	background-color: #58d162;
    border-radius: 3px;
}</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>621</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0">
       <item>
        <widget class="QPushButton" name="pushButton_Play_Stop">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color:  #91908d;
	width: 45px;
	height: 37px;
    border: 1px solid #000000;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>140</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_seek_Backwards">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color:  #91908d;
	width: 45px;
	height: 37px;
    border: 1px solid #000000;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_current_Time">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel
{
	color: white;
}</string>
         </property>
         <property name="text">
          <string>00:00:00,000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_seek_Forwards">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color:  #91908d;
	width: 45px;
	height: 37px;
    border: 1px solid #000000;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>10ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>200ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>500ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1000ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2000ms</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
     <item>
      <widget class="QLabel" name="label_current_Volume">
       <property name="styleSheet">
        <string notr="true">QLabel
{
	color: white;
}</string>
       </property>
       <property name="text">
        <string>Volume</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QSlider" name="verticalSlider_Volume">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::groove:vertical {
        background-color: #91908d;
        border: 1px solid #000000;
        width: 4px;
        border-radius: 3px;
}

QSlider::handle:vertical  {
	background-color: #58d162;
	border: 2px solid #515451;
	margin: 0 -7px;
	height: 5px;
	border-radius: 4px;
}

QSlider::add-page:vertical  {
	background-color:  #58d162;
    border-radius: 3px;
}

QSlider::sub-page:vertical  {
	background-color: #91908d;
    border-radius: 3px;
}</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>961</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Start_Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Stop_Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Sub_Text</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSubtitle">
    <property name="title">
     <string>Subtitle</string>
    </property>
    <addaction name="actionNew_Ctrl_N"/>
    <addaction name="actionOpen_Ctrl_O"/>
    <addaction name="actionSave_Ctrl_S"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Video</string>
    </property>
    <addaction name="actionOpen_Video_Ctrl_Alt_O"/>
    <addaction name="actionVideo_Info_Ctrl_Alt_I"/>
   </widget>
   <addaction name="menuSubtitle"/>
   <addaction name="menuVideo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Ctrl_N">
   <property name="text">
    <string>New Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_Ctrl_O">
   <property name="text">
    <string>Open Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Ctrl_S">
   <property name="text">
    <string>Save Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_Video_Ctrl_Alt_O">
   <property name="text">
    <string>Open Ctrl+Alt+O</string>
   </property>
  </action>
  <action name="actionVideo_Info_Ctrl_Alt_I">
   <property name="text">
    <string>Video Info Ctrl+Alt+I</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
